### ISSUE TITLE
Does MarkLight support TextMeshPro?

### ISSUE THREAD

**Issue by kokizzu** [delight-dev/Delight#3]
MarkLight reminded me to NativeScript, btw the font render seems jiggy? not smooth like when I used TextMeshPro
![image](https://user-images.githubusercontent.com/1061610/62894609-3b96b080-bd77-11e9-9b5d-3f335f4d523c.png)

**Comment by patrik-u** [comment]
Yes. Import the latest TextMeshPro through the package manager, and add the following to Config.txt:

```
Modules: 
  TextMeshPro 
```

As for the jiggy text, I'm not sure what the cause is. Might want to set PixelPerfect="True" on the root  UI canvas and see if that helps.
